database/data.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "1",
+    "productname": "Shirt",
```

Comment:
minor issue: maybe it would be nice to use camelCase or snake_case for naming -> makes it easier to read